10 timer add keyboard events to timer lab (#29)
comp129/comp55: docs/labs/10-Timer.md

### Second, create a numTimes instance variable

Let's create a private variable `numTimes`
that keeps track of how many times `actionPerformed` has been called

To do this in `run()`,
make sure to initialize that number to 0,
and increment `numTimes` at the start of `actionPerformed`.
Now just for fun,

> let's change the text of the label so that it updates the label with the ```numTimes```,

Reviewer: ```suggestion
let's change the text of the label so that it updates the label with the `numTimes`,
```

in making your programs more dynamic and expressive.

![label showing score](lab10media/media/image5.png)![game over score](lab10media/media/image6.png)

## Bonus 2

### *"Hey! Want to go further?"*

What if we want to create a ball launcher and
move the location from which the balls are being launched up and down?

> Adding Keyboard Listeners would be the perfect solution.

Reviewer: ```suggestion
Adding ***Keyboard Listeners*** would be the perfect solution.
```

These listeners are similar to Mouse Listeners and can detect and
respond to events such as key presses and releases.
By implementing the ```KeyListener``` interface
and adding ```addKeyListeners()``` to the ```run()```
method of your class, you can subscribe to keyboard event notifications.
After calling ```addKeyListeners()```, your program will be notified of all keyboard events
and you can interact with the program using the keyboard.
To get started, you can create a blue square on the left side of the screen
and use it as the launcher for the red balls.
Once you have added the launcher to the screen, you can easily move it using the keyboard keys.

> Similar to Mouse Listeners, you can define a method for KeyPressed that moves the launcher up or down

Reviewer: ```suggestion
Like Mouse Listeners, you can define a method for `keyPressed` that moves the launcher up or down
```
